requirements/common.txt

```diff
@@ -13,20 +10,15 @@ Flask-Script==2.0.5
 Flask-SQLAlchemy==2.0
 Flask-SSLify==0.1.5
 Flask-WTF==0.11
-geopy==1.11.0
```

Comment:
Isn't this used by the main site during production? `requirements/dev.txt` is (I think) meant to contain the requirements that are used only for development.